### PR TITLE
🐛 Fix google assistant capability

### PR DESCRIPTION
**Pull request by kouz75**
Link with #1257 
capabilities don't contains all informations

## Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

File: platforms/platform-googleassistant/src/GoogleAssistantDevice.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { GoogleAssistant } from './GoogleAssistant';
 import { Capability as NativeCapability } from './output';
 
 export enum GoogleAssistantCapability {
-  InteractiveCanvas = 'GOOGLE_ASSISTANT:INTERACTIVE_CANVAS',
```
Comment (jankoenig) [inline platforms/platform-googleassistant/src/GoogleAssistantDevice.ts]: Could you elaborate why you removed the ´GOOGLE_ASSISTANT` prefix? This is a convention we use for platform specific capabilities: https://www.jovo.tech/docs/device#platform-specific-device-features

**Comment by jankoenig** [comment]
Thank you @kouz75. 

I see that you linked this PR with #1257, but I can't find the missing capabilities in your committed code. Is there something missing?

**Comment by kouz75** [comment]
Hi 

I reverted my changes and fix the incorrect code.

**Review by aswetlow — APPROVED** [review]

**Review by jankoenig — APPROVED** [review]
Thank you @kouz75 🎉

**Pull request merged**